tests/src/contracts.ts

```diff
@@ -94,8 +94,8 @@ export interface ProxyOperationalMode {
 }
 
 export interface ProxyInstantiateMsg {
+  readonly manager: string;
```

Comment:
```suggestion
  readonly manager: null | string;
```

this makes it an Option